src/Datadog.Trace.ClrProfiler.Managed.Core/MethodReplacementActionType.cs

```diff
@@ -0,0 +1,27 @@
+namespace Datadog.Trace.ClrProfiler
+{
+    /// <summary>
+    /// Enum that instructs the CLR profiler, during JIT compilation of a method,
+    /// where to insert a method call to the intercept method.
+    /// </summary>
+    public enum MethodReplacementActionType
```

Comment:
Can we make this `internal`?

Reply:
If we set this to `internal` we won't be able to set the attribute when declaring the integration method. Since assembly can't be compiled against anyways (we don't provide it in a NuGet package), I think this should be fine as-is